recipes/r-fastcluster/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = '1.1.22' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-fastcluster
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: fastcluster_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/fastcluster_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/fastcluster/fastcluster_{{ version }}.tar.gz
+  sha256: 3f2d94490e349f738694b34e73614d813745b70a26328eb7f2711595754cd300
+
+build:
+  number: 0
+  script: R CMD INSTALL --build .
+  skip: True  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
```

Comment:
@bruening, thanks for all the reviews! Do you just mean that I need to add the line 
```yaml
  - libgcc  # [not win]
```
on line 34?